addon/components/flexberry-maptoolbar.js

```diff
@@ -11,9 +11,10 @@ export default Ember.Component.extend({
 
   activeTool: null,
 
-  actions: {
+  activeToolName: null,
```

Comment:
this property was removed by @kafnevod in their commits. Is it necessary for your code?